### PR TITLE
Update config.toml files

### PR DESCRIPTION
**Pull request by awaemmanuel**
- Remove the extra toml file in backend/app
- Update README instructions

### CODE REVIEW

File: infra/README.md [hunk 1]
```diff
@@ -106,16 +106,54 @@ In most Workspace setups, the Generative AI for Marketing application needs to b
...
+The configuration file acts as a control center for a marketing content generation, providing the necessary settings, prompts, and data to automate the creation of personalized and brand-consistent marketing materials.  
 
-Configuration for the backend of the application is stored in `backend_apis/app/config.toml`. Edit this file and update it with values generated during the deployment process. These instructions use `nano` but use whatever editor you prefer. From the `/infra` directory:
+The following are the key sections and their functions:
 
-1. Run `cat gdrive_folder_results.json` and copy the output to somewhere outside of your Cloud Shell window. You'll see configuration values for `drive_folder_id` , `slides_template_id`, `doc_template_id` and `sheet_template_id`. These will go into `config.toml`.
-1. Run `nano ../backend_apis/app/config.toml`
-1. Press ctrl+w to find `drive_folder_id` and you'll see where to enter the 4 configuration values.
-1. After updating the configuration values, press ctrl+x to exit then Y to save and enter to confim.
+#### **Global:**
```
Comment (michaelwsherman) [inline infra/README.md:115]: Move the config.toml overview to it's own section outside of the deployment guide.

**Review by michaelwsherman — COMMENTED** [review]
Please adjust the readme and make sure the end to end deployment steps still make sense. Config.toml description into root README.

File: infra/README.md (hunk 1 above)
Comment (awaemmanuel) [reply infra/README.md]: Done @michaelwsherman

**Review by michaelwsherman — APPROVED** [review]
LGTM

**Pull request merged**